cmd/backup-manager/app/export/manager.go

```diff
@@ -80,7 +87,16 @@ func (bm *BackupManager) ProcessBackup() error {
 		})
 	}
 
-	bm.setOptions(backup)
+	reason, err := bm.setOptions(backup)
+	if err != nil {
+		klog.Errorf("set mydumper backup %s option for cluster %s failed, err: %v", bm.ResourceName, bm, err)
+		return bm.StatusUpdater.Update(backup, &v1alpha1.BackupCondition{
```

Comment:
use `NewAggregate ` ?

Reply:
After this PR #2416 is merged, I will use `NewAggregate` instead.